.github/workflows/hadolint-lint.yml

```diff
@@ -0,0 +1,10 @@
+name: Hadolint Dockerfile Lint
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 
+      - uses: hadolint/hadolint-action@v2.0.0
+        with:
+          dockerfile: "cookiecutter-django-ida/\{\{cookiecutter.repo_name\}\}/Dockerfile"
```

Comment:
This doesn't work, I also tried {{, but that didn't work either: 
<img width="1103" alt="image" src="https://user-images.githubusercontent.com/278814/197880599-38b82ab0-f754-417a-9c27-0ec5c41ef24a.png">

Reply:
Even if you get that to work, the Dockerfile in question itself requires interpolation and has `{{` all over the place that will probably confuse a linter. Perhaps we can instead add the hadolint action to the cookiecutters that have a Dockerfile (xblock and ida) and also run hadolint from the cookiecutter repo's unit tests.

Reply:
I suspect the YAML parser is complaining about the `\{` since `{` does not require escaping. You say that there's a problem if you just use `{{` -- what complains? I'd expect GitHub to only complain if there's a `${{` sequence, but if it doesn't even like `{{` then you might have to do something gross like this:

    dockerfile: "cookiecutter-django-ida/${{ '{{' }}cookiecutter.repo_name${{ '}}' }}/Dockerfile"

However, I think you'll then just run into hadolint errors.